app/poros/movie_details.rb

```diff
@@ -0,0 +1,58 @@
+class MovieDetails
+  attr_reader :title,
+              :vote_average,
+              :runtime,
+              :genres,
+              :description,
+              :cast_members,
+              :reviews,
+              :id
+
+  def initialize(details)
+    @title = details[:movie_details][:original_title]
+    @vote_average = details[:movie_details][:vote_average]
+    @runtime = format_runtime(details[:movie_details][:runtime])
+    @genres = format_genre(details[:movie_details][:genres])
+    @description = details[:movie_details][:overview]
+    @cast_members = format_cast_members(details[:cast_details])
+    @reviews = format_reviews(details[:reviews][:results])
+    @id = details[:movie_details][:id]
+  end
+
+  def format_genre(genre_array)
```

Comment:
I wonder if this can be reformatted with a .map or even a .reduce?

Reply:
Ooh - now I see this logic is repeated in methods below - I wonder if it can be a single helper method (or even a Module)